src/pageObjects/components/SignUpForm.js

```diff
@@ -0,0 +1,74 @@
+import BaseComponent from "../BaseComponent.js";
+import {expect} from "@playwright/test";
+
+export default class SignUpForm extends BaseComponent {
+
+    _inputName = '#signupName';
+    _inputLastName = '#signupLastName';
+    _inputEmail = '#signupEmail';
+    _inputPassword = '#signupPassword';
+    _inputRepeatPassword = '#signupRepeatPassword';
+    _registerButton = '.modal-footer button';
+    _alertDanger = '.alert-danger';
+    _errors = '.invalid-feedback';
+
+    constructor(page, inputValues) {
+        super(page, page.locator('div.modal-content'));
+        this.inputValues = inputValues;
```

Comment:
this input values have no relation to page. It is bad idea to store it in the page object. Pass this values when you fill form